test/e2e/cronjob_test.go

```diff
@@ -56,7 +59,57 @@ func TestSourceCronJob(t *testing.T) {
 	jpSinkRefNameInSpec := "jsonpath={.spec.sink.ref.name}"
 	out, err := test.getResourceFieldsWithJSONPath("cronjobsource", "testcronjobsource2", jpSinkRefNameInSpec)
 	assert.NilError(t, err)
-	assert.Equal(t, out, "testsvc1")
+	//assert.Equal(t, out, "testsvc1")
```

Comment:
please remove

Reply:
> @daisy-ycguo with the latest commits, there is a conflict.
> 
> wrt/ the issue: I will open an issue on test-infrastructure so that the istio local gateway gets enabled for the tests.

Until test-infrastructure gets update, this PR will not pass the integration test.